### PR TITLE
Upgrade the app and dependencies

### LINKED ISSUE #3: runtime org.freedesktop.Platform branch 20.08 is end-of-life
org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.

### PR DESCRIPTION
**Pull request by TomaszGasior**
Fixes #3

### CODE REVIEW

**Comment by TomaszGasior** [comment]
Waiting for feedback in https://github.com/danielknobe/blobbyvolley2/issues/15#issuecomment-1650420467

File: de.blobbyvolley.BlobbyVolley2.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 app-id: de.blobbyvolley.BlobbyVolley2
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
```
Comment (bbhtt) [inline de.blobbyvolley.BlobbyVolley2.yaml]: Can you try 23.08? Thanks

File: de.blobbyvolley.BlobbyVolley2.yaml [hunk 2]
```diff
@@ -22,45 +25,25 @@ modules:
...
-      - install -D de.blobbyvolley.BlobbyVolley2.desktop /app/share/applications/de.blobbyvolley.BlobbyVolley2.desktop
-      - install -D de.blobbyvolley.BlobbyVolley2.metainfo.xml /app/share/metainfo/de.blobbyvolley.BlobbyVolley2.metainfo.xml
-    sources:
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.png
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.desktop
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.metainfo.xml
+        tag: v1.1
```
Comment (bbhtt) [inline de.blobbyvolley.BlobbyVolley2.yaml]: 1.1.1 is out and looks like upstream won't support migrating https://github.com/danielknobe/blobbyvolley2/issues/15#issuecomment-1650486768

**Comment by TomaszGasior** [comment]
@bbhtt Thank you for your comments. I will take a look on them in next few days.

**Pull request merged**